inference-engine/src/mkldnn_plugin/nodes/detectionoutput.cpp

```diff
@@ -243,6 +281,7 @@ class DetectionOutputImpl: public ExtLayerBase {
                     detections_data[n*_num_classes + label]++;
                 }
             }
+            detections_batch[n] = detections_total;
```

Comment:
You are doing several changes in the code that relate not only 5 inputs support but also change old behavior (like with 3 inputs). Why? Does it means that previous implementation was incorrect?

Reply:
previous implementation ```nms_mx``` work incorrect with ```share_location == false``` (incorrect compute offset for ```bboxes```